docs/developer-guide/syntax-reference.rst

```diff
@@ -1734,6 +1739,10 @@ Scalar functions
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
 | RANDOM                 |  ``RANDOM()``                                                             | Return a random DOUBLE value between 0.0 and 1.0. |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| RANGE                  |  ``RANGE(start_inclusive, end_exclusive)``                                | Constructs an array of values between             |
+|                        |                                                                           |`start_inclusive` and `end_exclusive` (exclusive). |
```

Comment:
`start_inclusive` and `end_exclusive` need double backticks, which will require reformatting the table (sadly).

Reply:
done